cdap-data-fabric/src/main/java/co/cask/cdap/store/StoreDefinition.java

```diff
@@ -601,4 +605,62 @@ public static void createTables(StructuredTableAdmin tableAdmin) throws IOExcept
       tableAdmin.create(MODULE_TYPES_SPEC);
     }
   }
+
+  /**
+   * Schema for {@link LineageTable}.
+   */
+  public static final class LineageStore {
+    public static final StructuredTableId DATASET_LINEAGE_TABLE = new StructuredTableId("dataset_lineage");
+    public static final StructuredTableId PROGRAM_LINEAGE_TABLE = new StructuredTableId("program_lineage");
+    public static final String NAMESPACE_FIELD = "namespace";
+    public static final String DATASET_FIELD = "dataset";
+    public static final String START_TIME_FIELD = "start_time";
+    public static final String PROGRAM_NAMESPACE_FIELD = "program_namespace";
+    public static final String PROGRAM_PARENT_FIELD = "program_parent";
```

Comment:
this should be PROGRAM_APPLICATION_FIELD and program_app

Reply:
also (see my comment above) where is the program version recorded?

Reply:
unless I am mistaken it was never recorded?

Reply:
oh bummer. So this was never there. Or maybe on purpose? In that case, ok.